src/palace/manager/marc/settings.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+from pydantic import NonNegativeInt
+
+from palace.manager.integration.settings import (
+    BaseSettings,
+    ConfigurationFormItem,
+    ConfigurationFormItemType,
+    FormField,
+)
+
+
+class MarcExporterSettings(BaseSettings):
+    # This setting (in days) controls how often MARC files should be
+    # automatically updated. Since the crontab in docker isn't easily
```

Comment:
The comment should be updated to reflect that we're not using crontab anymore.

Reply:
Good catch, thanks!